addon-test-support/-private/is-focusable.js

```diff
@@ -0,0 +1,10 @@
+export default function isFocusable(el) {
```

Comment:
Do you need this outside the `focus.js` file? It could just be a function there.